src/app/workflow/workflow.component.ts

```diff
@@ -390,39 +390,41 @@ export class WorkflowComponent extends Entry<WorkflowVersion> implements AfterVi
   }
 
   public setupPublicEntry(url: string) {
-    const subclass: WorkflowSubClass = this.getWorkflowSubclass(this.entryType);
-    this.workflowsService
-      .getPublishedWorkflowByPath(this.title, subclass, includesValidation + ',' + includesAuthors, this.urlVersion)
-      .subscribe(
-        (workflow) => {
-          this.workflowService.setWorkflow(workflow);
-          this.selectTab(this.validTabs.indexOf(this.currentTab));
-          this.updateWorkflowUrl(this.workflow);
-        },
-        (error) => {
-          const workflowOrgRegex = /\/workflows\/.+/;
-          const workflowRegex = /\/workflows\/(github.com)|(gitlab.com)|(bitbucket.org)\/.+/;
-          const gitHubAppToolRegex = /\/containers\/(github.com)\/.+/;
-          if (
-            workflowOrgRegex.test(this.resourcePath) ||
-            workflowRegex.test(this.resourcePath) ||
-            gitHubAppToolRegex.test(this.resourcePath)
-          ) {
-            this.router.navigate(['page-not-found']);
-          } else {
-            this.showRedirect = true;
-            // Retrieve the workflow path from the URL
-            const splitPath = this.resourcePath.split('/');
-            const workflowPath = splitPath.slice(2, 5);
-            const pathSuffix = workflowPath.join('/');
-
-            // Create suggested paths
-            this.gitlabPath += pathSuffix;
-            this.githubPath += pathSuffix;
-            this.bitbucketPath += pathSuffix;
+    if (!url.includes('containers') && !url.includes('tools')) {
```

Comment:
I added this line so that the request isn't made on a legacy tool page. The integration and smoke tests pass but I wonder if this would prevent the request from being made on any pages it's actually needed.